web/components/FcDrawerViewCollisionReports.vue

```diff
@@ -26,7 +26,42 @@
           <h1 class="headline ml-4">Collisions</h1>
           <div
             class="ml-1 font-weight-regular headline secondary--text">
-            <span>&#x2022; {{locationsDescription}}</span>
+            <span>&#x2022;</span>
+            <span v-if="locationMode === LocationMode.SINGLE">
+              {{locationActive.description}}
+            </span>
+            <span v-else-if="!detailView">
+              {{locationsDescription}}
+            </span>
+            <v-menu
+              v-else
+              max-height="320">
+              <template v-slot:activator="{ on, attrs }">
+                <FcButton
+                  v-bind="attrs"
+                  v-on="on"
+                  class="flex-grow-0 mt-0 ml-2"
+                  type="secondary">
+                  <FcIconLocationMulti v-bind="locationsIconProps[locationsIndex]" />
```

Comment:
we use this "location list" pattern in a bunch of places - I've submitted #589 to clean that up when we get time.